### ISSUE TITLE
Pause/run Growfile automation

### ISSUE THREAD

**Issue by catbarph** [CommonGarden/Grow-IoT#432]
During setup and in use, it is helpful to pause/resume the growfile/driver at times.

**Comment by JakeHartnell** [comment]
Might as well add the ability to transition to the next phase in the growfile too.

**Comment by catbarph** [comment]
Indeed, a different (graphical?) representation of the phases of the growfile would be nice... 

However, @JakeHartnell I'm mainly talking about pausing the driver so that a sensor or actuator or the like can be safely switched out, without unknown/untested conditions getting into the mix and throwing off the PID loop and so on.

**Comment by catbarph** [comment]
Resolved with web-UI switch?
